File: Programmers_cs/두 정수 사이의 합.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApplication1
{
    class Program
    {
        static void Main(string[] args)
        {
            Solution so = new Solution();
            Console.WriteLine(so.solution(3,5));
        }
    }

    public class Solution
    {
        public int solution(int a, int b)
        {
                int sum = 0;
                if (a>b) {
                    for(int i=a; i>=b; i--) sum+=i; 
                } else {
                    for(int i=a; i<=b; i++) sum+=i; 
                }
                int answer = sum;
                return answer;
        }
    }
}

File: Programmers_cpp/Level1/두_정수_사이의_합.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

long long solution(int a, int b) {
	long sum = 0;
	if (a>b) {
		for (int i = a; i >= b; i--) sum += i;
	}
	else {
		for (int i = a; i <= b; i++) sum += i;
	}
	long long answer = sum;
	return answer;
}

int main(void) {
	int answer = solution(-1,1);

	cout << answer;
  }
}
